packages/sdk/src/types/runtime/assets/texture.ts

```diff
@@ -28,6 +29,7 @@ export enum TextureWrapMode {
 
 export class Texture extends Asset implements TextureLike, Patchable<AssetLike> {
     // tslint:disable:variable-name
+    private _uri: string;

```

Comment:
>_uri [](start = 12, length = 4)

If this value is observed, it needs a default value to be assigned before the observe hooks are set. Otherwise, it won't be hooked (because it doesn't exist on the object until assigned).

Reply:
It's not observed. In fact it doesn't have a setter app-side. It's assumed that this value is only set host-side.